### DISCUSSION TITLE
So what happened to bare YAML installs?

### DISCUSSION THREAD

**Discussion by ViliusS** [kubernetes/dashboard#8759] [Q&A]
Newest dashboard release doesn't have all-in-one YAML files to install it. I don't see anything mentioned in the release notes about deprecation of this install method either. I'm confused. Will it be available some time later or were YAML files as an install method abandoned?

**Comment by floreks** [comment]
Indeed we might have forgotten to mention everything in the release notes. We did update the main README though. Unfortunately, due to the complexity of the current setup, it would be very hard to support manifest-based installation. We leverage helm features quite heavily, i.e. to generate CSRF key on the fly and store it in secret during the installation.

Dependency handling is also a pain when it comes to raw YAML installation. At this point, the whole setup is too complex to be able to easily provide this option. I don't see how we could support that.

**Reply by ViliusS** [reply]
I think it's still not too late to edit Release Notes at least for 7.0.0 on GitHub and just add one more sentence, that anyone looking to upgrade their YAML installation should now use Helm Charts. Other projects often release Helm charts and YAML files at different cadence, so for anyone tracking releases on GitHub it could be not very obvious, that the old method is actually gone.

**Reply by floreks** [reply]
@ViliusS I have updated our README to cover this. Hopefully, that is enough. https://github.com/kubernetes/dashboard/blob/master/README.md#introduction
